lib/ethui/services/anvil.ex

```diff
@@ -95,37 +99,46 @@ defmodule Ethui.Services.Anvil do
          dir: dir,
          slug: opts[:slug],
          log_subscribers: MapSet.new(),
-         chain_id: chain_id()
+         chain_id: chain_id(),
+         opts: opts_to_args(opts[:anvil_opts])
```

Comment:
if it's `opt_to_args` then the `opts` keys in the final map should be renamed to `args` for clarity, no?

Reply:
yup that would make a lot more sense !